app/src/main/java/org/apache/taverna/mobile/data/remote/BaseApiManager.java

```diff
@@ -20,27 +20,42 @@
 
 import retrofit2.Retrofit;
 import retrofit2.adapter.rxjava.RxJavaCallAdapterFactory;
+import retrofit2.converter.gson.GsonConverterFactory;
 import retrofit2.converter.simplexml.SimpleXmlConverterFactory;
 
 
 public class BaseApiManager {
 
 
-    public static final String END_POINT = "http://www.myexperiment.org/";
-    public TavernaService mTavernaService;
+    public static final String MY_EXPERIMENT_END_POINT = "http://www.myexperiment.org/";
 
-    public BaseApiManager() {
+    public static final String TAVERNA_PLAYER_END_POINT = "http://139.59.28.12:3000/";
```

Comment:
I know it is useful for testing but might not be a good idea to distribute code with it hard coded.